### PR TITLE
Add prop `isOpen` to Navigation

### PR DESCRIPTION
**Pull request by Bopchy**
Resolves #NUMBER

**Overall change:** _Adds a prop `isOpen` that if true, changes the background colour of the Navigation from C_POSTBOX to C_EBON._

**Code changes:**

- _Add `isOpen` prop._
---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Automated jest tests added (for new features) or updated (for existing features)
- [ ] This PR requires manual testing

### CODE REVIEW

**Comment by ghost** [comment]
How do i view this change?

**Comment by Bopchy** [comment]
Currently this is not viewable in Psammead as the colour of the Navigation should change when the Dropdown is open - and the components have been put together in Simorgh. 

This is what the Navigation currently looks like when the menu is open
image

**Comment by Bopchy** [comment]
Renamed the PR to reflect the changes it is adding better

File: packages/components/psammead-navigation/README.md [hunk 1]
```diff
@@ -24,8 +24,9 @@ The `@bbc/psammead-navigation` package is a set of two components, `NavigationUl
 <!-- prettier-ignore -->
 | Argument | Type | Required | Default | Example |
 | -------- | ---- | -------- | ------- | ------- |
-| children | node | Yes      | N/A     | `<ScrollableNavigation dir={dir}><NavigationUl><NavigationLi url="/" script={latin} active="true">Home</NavigationLi><NavigationLi url="/sport" script={latin}>{Sport}</NavigationLi></NavigationUl><ScrollableNavigation/>` |
-| dir      | string  | No       | `'ltr'`   | `'rtl'` |
+| children | node | Yes | N/A | `<ScrollableNavigation dir={dir}><NavigationUl><NavigationLi url="/" script={latin} active="true">Home</NavigationLi><NavigationLi url="/sport" script={latin}>{Sport}</NavigationLi></NavigationUl><ScrollableNavigation/>` |
+| dir | string | No | `'ltr'` | `'rtl'` |
+| changeBackground | boolean | No | `false` | `true` |
```
Comment (ghost) [inline packages/components/psammead-navigation/README.md]:
```suggestion
| isOpen | boolean | No | `false` | `true` |
```

File: packages/components/psammead-navigation/CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 |---------|-------------|
+| 6.0.0-alpha.6 | [PR#2769](https://github.com/bbc/psammead/pull/2769) Add `changeBackground` prop to `Navigation` |
```
Comment (OlgaLyubin) [inline packages/components/psammead-navigation/CHANGELOG.md]:
```suggestion
| 6.0.0-alpha.6 | [PR#2769](https://github.com/bbc/psammead/pull/2769) Add `isOpen` prop to `Navigation` |
```

File: packages/components/psammead-navigation/src/index.jsx [hunk 3]
```diff
@@ -214,7 +219,8 @@ NavigationLi.defaultProps = {
 
 const StyledNav = styled.nav`
   position: relative;
-  background-color: ${C_POSTBOX};
+  ${({ isOpen }) =>
+    isOpen ? `background-color: ${C_EBON};` : `background-color: ${C_POSTBOX};`}
```
Comment (ghost) [inline packages/components/psammead-navigation/src/index.jsx]: Only the colors value is changing. May be we could do.
```js
background-color:   isOpen ? `${C_EBON}` : `${C_POSTBOX}`}
```

Comment (Bopchy) [reply packages/components/psammead-navigation/src/index.jsx]: Done.

**Review by ghost — COMMENTED** [review]
This looks great. We could add a navigation snapshot test when the `isOpen` prop is `true`.

**Review by oluoluoxenfree — APPROVED** [review]

**Comment by Bopchy** [comment]
Added a snapshot test.

**Review by AlistairGempf — APPROVED** [review]
Looks good! Nicely done.

File: packages/components/psammead-navigation/src/index.test.jsx [hunk 4]
```diff
@@ -9,33 +9,48 @@ import {
 import Navigation, { NavigationUl, NavigationLi } from './index';
 import igboNavData from '../testHelpers/igbo';
 
+const NavigationData = (
```
Comment (ghost) [inline packages/components/psammead-navigation/src/index.test.jsx]: What do you think about renaming `NavigationData` to something like `navigationUlComponent`
```jsx
<Navigation>
  {NavigationData} /* gives an impression that we are rendering a json object. */
</Navigation>
```

Comment (Bopchy) [reply packages/components/psammead-navigation/src/index.test.jsx]: I think it makes sense. Done!

**Review by j-pendlebury — APPROVED** [review]
lgtm 👍

**Review by ghost — APPROVED** [review]

**Pull request merged**